packages/database/src/migrations/20210705232632-AddECEGoalsDrillDowns-modifies-data.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord, findSingleRecordBySql } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const generateReport = (dataElement, target) => ({
+  fetch: {
+    dataElements: [dataElement],
+    aggregations: [
+      {
+        type: 'MOST_RECENT',
+      },
+      {
+        type: 'SUM_PER_ORG_GROUP',
+        config: {
+          dataSourceEntityType: 'sub_district',
+          aggregationEntityType: 'requested',
+        },
+      },
+    ],
+  },
+  transform: [
+    'keyValueByDataElementName',
+    {
+      transform: 'select',
+      "'Current'": `$row.${dataElement}/100`,
+      "'Target'": `${target || undefined}`,
```

Comment:
This will need to be changed if we swap these targets to being data elements like mentioned in [this comment](https://github.com/beyondessential/tupaia-backlog/issues/2984#issuecomment-875947356)

Reply:
We now have them in indicators, so time to look at this again I think!

Reply:
This gets swapped to use indicators in the [other PR](https://github.com/beyondessential/tupaia/pull/2878/files#diff-c2b3997ded812179eabbc50d322c6613a97e97e87025ee47806d28d1c399a074R62)

Reply:
Oh of course, I just reviewed that 🤦